python/ray/tests/accelerators/test_amd_gpu.py

```diff
@@ -79,6 +79,18 @@ def test_get_current_process_visible_accelerator_ids(
     )
 
 
+def test_hip_cuda_env_var_get_current_process_visible_accelerator_ids(monkeypatch):
+    monkeypatch.setenv("HIP_VISIBLE_DEVICES", "0,1,2")
+    monkeypatch.setenv("CUDA_VISIBLE_DEVICES", "0,1,2")
+    assert AMDGPUAcceleratorManager.get_current_process_visible_accelerator_ids() == [
+        "0",
+        "1",
+        "2",
+    ]
+    del os.environ["HIP_VISIBLE_DEVICES"]
```

Comment:
No need to manually del, monkeypatch will restore the env var after test finishes.